operating/view-dashboard.adoc

```diff
@@ -6,58 +7,100 @@ include::modules/common-attributes.adoc[]
+The top of the dashboard shows an overview of system data, including the number of clusters, nodes, violations, deployments, images, and secrets. Choosing one of those items takes you to the main menu page for the item. For example, select *Deployments* to view the *Configuration Management* -> *Deployments* page.
 
-[id="compliance"]
-== Compliance
+[id="select-resources"]
+== Selecting resources for data display
 
-The *Compliance* widget (upper-right) shows the status of various compliance benchmarks and if their controls are passing or failing.
+You can review metrics for your entire system or only selected resources by selecting the following options:
 
-[NOTE]
-====
-The *Compliance* widget show details only after you run a compliance scan.
-====
-//TODO: Add link to compliance scan
+* To view data for specific clusters, select the clusters from the *Resources* drop-down list. 
+* To view data for all clusters, select *All clusters*. 
+* To view data for specific namespaces, select them from the *Resources* drop-down list. Note that all namespaces are automatically included when you select *All clusters*. 
+
+[id="policy-violations-by-severity"]
+== Policy violations by severity
+
+The *Policy violations by severity* widget displays the total number of violations and displays the number of violations by severity level. Severity levels are color-coded with gray for low severity; yellow for medium severity; orange for high severity; and red for critical severity. 
+
+You can select items to view additional information as follows:
+
+* To view all violations of a specific severity, click the count for the desired severity level. 
+* To view all violations of any severity, select *View all*. Choosing a violations category or *View all* displays the *Violations* page and the menu bar reflects any *Cluster* or *Namespace* selections that you made in the dashboard.
+
+The *Policy violations by severity* widget also displays the three most recent violations that have a severity level of *Critical*. Choosing a specific violation displays the xref:../operating/respond-to-violations.adoc#view-violation-details[Violations detail] page that provides additional information about the violation.
+
+[id="images-at-most-risk"]
+== Images at most risk
+
+The *Images at most risk* widget lists images that have the highest risk score. Risk score is determined by an analysis that uses number of critical common vulnerabilities and exposures (CVEs), severity of CVEs, and other factors.
+
+You can configure items for display in the widget by selecting the following options:
+
+* *Image vulnerabilities*:
+**  *Fixable CVEs*: Displays the number of fixable CVEs in the *Critical CVEs* and *Important CVEs* columns.
+**  *All CVEs*: Displays the total number of CVEs in the *Critical CVEs* and *Important CVEs* columns.
+* *Image status*: 
+** *Active images*: Displays the active images at most risk. 
+** *All images*: Displays all images at most risk.
```

Comment:
"all images at most risk" sounds off I think. Maybe:  Displays images at most risk, regardless of their active/inactive status.  @dvail @vjwilson thoughts?

Reply:
We made a similar change in the UI based on feedback. Originally the widget title was "All images at most risk" but it was changed right before release to "Images at most risk".